deepchecks/utils/distribution/plot.py

```diff
@@ -108,6 +108,16 @@ def drift_score_bar_traces(drift_score: float, bar_max: float = None) -> Tuple[L
     return bars, xaxis, yaxis
 
 
+def _un_numpy(val):
```

Comment:
Can you please add a comment with a link to the issue and some elaborations?

Reply:
did a comment